common-pg/src/main/scala/com/socrata/pg/store/PGSecondaryUniverse.scala

```diff
@@ -73,7 +73,7 @@ class PGSecondaryUniverse[SoQLType, SoQLValue](
   }
 
   def schemaLoader(logger: Logger[SoQLType, SoQLValue]): SecondarySchemaLoader[SoQLType,SoQLValue] =
-    new SecondarySchemaLoader(conn, logger, repForIndex, tablespace, commonSupport, RowSizeBufferSqlErrorContinue)
+    new SecondarySchemaLoader(conn, logger, repForIndex, tablespace, commonSupport, RowSizeBufferSqlErrorContinue, timingReport)
```

Comment:
Where does the variable `timingReport` come from in this file?

Reply:
timingReport came from the caller PGSecondaryUniverse